docs/data-sources/storage.md

```diff
@@ -219,7 +219,7 @@ Read-Only:
 - `dell_controller_battery` (Attributes) dell controller battery (see [below for nested schema](#nestedatt--storage--oem--dell--dell_controller_battery))
 
 <a id="nestedatt--storage--oem--dell--dell_controller"></a>
-### Nested Schema for `storage.oem.dell.dell_controller_battery`
+### Nested Schema for `storage.oem.dell.dell_controller`
 
 Read-Only:
 
```

Comment:
@taohe1012 Curious why there are so many doc updates? They were wrong and now we do the correction?

Reply:
@forrestxia Before the terraform-plugin-docs version is v0.16.0.
Now these two doc files are generated by terraform-plugin-docs@v0.19.4. And the new generated doc are correct.